desc/objectives/_bootstrap.py

```diff
@@ -31,31 +31,6 @@ class BootstrapRedlConsistency(_Objective):
     ----------
     eq : Equilibrium
         Equilibrium that will be optimized to satisfy the Objective.
-    target : {float, ndarray}, optional
-        Target value(s) of the objective. Only used if bounds is None.
-        Must be broadcastable to Objective.dim_f. Defaults to ``target=0``.
```

Comment:
need to override this and bounds to include default target. (it might be better to add default target/bounds as args to collect_docs since each objective will have to override this)

Reply:
need this in several other places too